src/main/i18n/mucextinfo_i18n_en.properties

```diff
@@ -1,7 +1,7 @@
 settings.saved.successfully=Settings Saved.
 admin.error=Error
 global.csrf.failed=CSRF security check failed. Please reload the page and try again.
-sidebar.muc-room-extinfo-edit-form=Service Disovery Ext.
+sidebar.muc-room-extinfo-edit-form=Service Discovery Ext.
 sidebar.muc-room-extinfo-edit-form.descr=Click to edit the room's extended service discovery information.
```

Comment:
Why aren't properties files using "double apostrophes" to represent an apostrophe? Isn't that the norm here?

Reply:
Good catch! I didn't know we even had an issue with those. I do see the apostrophe show with a leading slash - that's no good (hard to create a screenshot, as it's a tooltip). I'll replace it with what's used in the 'room occupants' tab: a HTML encoded character.